### DISCUSSION TITLE
Each photo is multiplied after it is uploaded

### DISCUSSION THREAD

**Discussion by crstian19** [LycheeOrg/Lychee#2663] [Bugs] [answered]
### Lychee version

v6.0.1

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

When I upload photos to an album lychee multiplies them and uploads the same photo 2,3,4 times.

I have tried in different albums and it keeps happening.

![image](https://github.com/user-attachments/assets/c8995c1d-0485-4746-8a0f-73f4a984d660)
![image](https://github.com/user-attachments/assets/1b61ec35-9ff0-4591-8bd5-0f3b2fc465eb)
![image](https://github.com/user-attachments/assets/f967bcff-41fd-4a7f-bd13-9fa9f64ceb05)

### Steps to reproduce the issue

Upload multiple photos

### Diagnostics [REQUIRED]

Info

Lychee Version (tag):                      v6.0.1 (6fc3e35) -- Up to date (-1.1574074074074E-10 years ago).
DB Version:                                6.0.1
composer install:                          --no-dev
APP_ENV:                                   dev
APP_DEBUG:                                 true
APP_URL:                                   set
APP_DIR:                                   default
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true
System:                                    Linux
PHP Version:                               8.2.24
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  UTC
Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                25.60 MB
Max execution time:                        3600
PostgreSQL Version:                        PostgreSQL 17.0 on x86_64-pc-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1691
GD Version:                                2.3.3
Number of foreign key:                     12 found.

		

Space

Lychee total space:                        219.47 MB
Upload folder space:                       8.00 B
System total space:                        1.82 TB
System free space:                         1.30 TB (71%)

		

Configuration

version:                                   060001
check_for_updates:                         0
sorting_photos_col:                        taken_at
sorting_photos_order:                      ASC
sorting_albums_col:                        max_taken_at
sorting_albums_order:                      DESC
imagick:                                   1
skip_duplicates:                           1
small_max_width:                           0
small_max_height:                          360
medium_max_width:                          1920
medium_max_height:                         1080
lang:                                      en
image_overlay_type:                        exif
default_license:                           none
compression_quality:                       90
grants_full_photo_access:                  1
delete_imported:                           0
mod_frame_enabled:                         1
mod_frame_refresh:                         30
thumb_2x:                                  1
small_2x:                                  1
medium_2x:                                 1
landing_page_enable:                       0
force_migration_in_production:             0
search_public:                             0
hide_version_number:                       0
SL_enable:                                 0
SL_for_admin:                              0
recent_age:                                1
grants_download:                           0
photos_wraparound:                         1
map_display:                               1
zip64:                                     1
map_display_public:                        0
map_provider:                              Wikimedia
force_32bit_ids:                           0
map_include_subalbums:                     1
update_check_every_days:                   3
has_exiftool:                              1
share_button_visible:                      0
import_via_symlink:                        0
has_ffmpeg:                                1
location_decoding:                         1
location_decoding_timeout:                 30
location_show:                             1
location_show_public:                      0
rss_enable:                                0
rss_recent_days:                           7
rss_max_items:                             100
prefer_available_xmp_metadata:             0
editor_enabled:                            1
lossless_optimization:                     0
swipe_tolerance_x:                         150
swipe_tolerance_y:                         250
local_takestamp_video_formats:             .avi|.mov
log_max_num_line:                          1000
unlock_password_photos_with_url_param:     0
nsfw_visible:                              1
nsfw_blur:                                 0
nsfw_warning:                              0
nsfw_warning_admin:                        0
nsfw_banner_override:                      
map_display_direction:                     1
album_subtitle_type:                       oldstyle
upload_processing_limit:                   4
new_photos_notification:                   0
legacy_id_redirection:                     1
zip_deflate_level:                         6
default_album_protection:                  1
allow_username_change:                     1
album_decoration:                          layers
album_decoration_orientation:              row
auto_fix_orientation:                      1
use_job_queues:                            0
random_album_id:                           starred
use_last_modified_date_when_no_exif_date:  0
layout:                                    justified
date_format_photo_thumb:                   M j, Y, g:i:s A e
date_format_photo_overlay:                 M j, Y, g:i:s A e
date_format_sidebar_uploaded:              M j, Y, g:i:s A e
date_format_sidebar_taken_at:              M j, Y, g:i:s A e
date_format_hero_min_max:                  F Y
date_format_hero_created_at:               M j, Y, g:i:s A T
date_format_album_thumb:                   M Y
upload_chunk_size:                         0
nsfw_banner_blur_backdrop:                 0
search_pagination_limit:                   1000
search_minimum_length_required:            4
photo_layout_justified_row_height:         320
photo_layout_masonry_column_width:         300
photo_layout_grid_column_width:            250
photo_layout_square_column_width:          200
photo_layout_gap:                          12
display_thumb_album_overlay:               always
display_thumb_photo_overlay:               hover
default_album_thumb_aspect_ratio:          1/1
use_album_compact_header:                  0
login_button_position:                     left
back_button_enabled:                       0
back_button_text:                          Return to Home
current_job_processing_visible:            1
SA_random_thumbs:                          0
job_processing_queue_visible:              1
login_required:                            0
enable_unsorted:                           1
enable_starred:                            1
enable_recent:                             1
enable_on_this_day:                        1
login_required_root_only:                  1
dark_mode_enabled:                         1
show_keybinding_help_popup:                0
show_keybinding_help_button:               1
disable_se_call_for_actions:               0
enable_se_preview:                         0
slideshow_timeout:                         5
keep_original_untouched:                   1
hide_nsfw_in_smart_albums_and_search:      1

		

### Browser & System [REQUIRED]

NixOS 24.11 with Docker

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
https://github.com/LycheeOrg/Lychee/discussions/2611
Already aware.

I see that you are using `tag`, can I trouble you to test latest `master` ? 

I have some hopes that https://github.com/LycheeOrg/Lychee/commit/2001c44838af7e33aa5bf77b874706456cf31e67 fixed it.

**Reply by crstian19** [reply]
I tried latest and nightly version and still happen

**Reply by ildyria** [reply]
thanks for the feedback :(

**Reply by ildyria** [reply]
that one is very nasty to track. If you go to an album, refresh the page, it does not trigger. :(
